### PR TITLE
[NUI] Refactoring FrameUpdateCallback

### PR DESCRIPTION
**Pull request by bshsqa**
- Remove FrameCallback.cs and Interop.FrameCallback.cs
 - Rename FrameCallbackInterface to FrameUpdateCallbackInterface

### CODE REVIEW

File: src/Tizen.NUI/src/internal/Interop/Interop.FrameUpdateCallbackInterface.cs [hunk 1]
```diff
@@ -6,13 +6,13 @@ namespace Tizen.NUI
 {
     internal static partial class Interop
     {
-        internal static partial class FrameCallbackInterface
+        internal static partial class FrameUpdateCallbackInterface
         {
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_new_FrameCallbackInterface")]
-            public static extern global::System.IntPtr new_FrameCallbackInterface();
+            public static extern global::System.IntPtr new_FrameUpdateCallbackInterface();
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/internal/Interop/Interop.FrameUpdateCallbackInterface.cs:12]: if possible, could you remove this "\_" in name

File: src/Tizen.NUI/src/internal/Interop/Interop.FrameUpdateCallbackInterface.cs [hunk 2]
```diff
@@ -6,13 +6,13 @@ namespace Tizen.NUI
...
 
             [global::System.Runtime.InteropServices.DllImport(NDalicPINVOKE.Lib, EntryPoint = "CSharp_Dali_FrameCallbackInterface_director_connect")]
-            public static extern void FrameCallbackInterface_director_connect(global::System.Runtime.InteropServices.HandleRef jarg1, Tizen.NUI.FrameCallbackInterface.SwigDelegateFrameCallbackInterface delegate0);
+            public static extern void FrameUpdateCallbackInterface_director_connect(global::System.Runtime.InteropServices.HandleRef jarg1, Tizen.NUI.FrameUpdateCallbackInterface.SwigDelegateFrameUpdateCallbackInterface delegate0);
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/internal/Interop/Interop.FrameUpdateCallbackInterface.cs:15]: if possible, could you remove this "_" in name
and also if possible, please remove "Swig" word itself~

File: src/Tizen.NUI/src/public/FrameUpdateCallbackInterface.cs [hunk 3]
```diff
@@ -26,31 +26,31 @@ namespace Tizen.NUI
...
+        internal static global::System.Runtime.InteropServices.HandleRef getCPtr(FrameUpdateCallbackInterface obj)
         {
             return (obj == null) ? new global::System.Runtime.InteropServices.HandleRef(null, global::System.IntPtr.Zero) : obj.swigCPtr;
         }
 
         private void SwigDirectorConnect()
         {
-            swigDelegate0 = new SwigDelegateFrameCallbackInterface(SwigDirectorOnUpdate);
-            Interop.FrameCallbackInterface.FrameCallbackInterface_director_connect(swigCPtr, swigDelegate0);
+            swigDelegate0 = new SwigDelegateFrameUpdateCallbackInterface(SwigDirectorOnUpdate);
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/FrameUpdateCallbackInterface.cs:52]: same as above

File: test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/FrameUpdateCallbackTest.cs [hunk 4]
```diff
@@ -368,7 +368,7 @@ public override void OnUpdate(float elapsedSeconds)
 
     public void Activate()
     {
-      mFrameCallback = new FrameCallback();
+      mFrameUpdateCallback = new FrameUpdateCallback();
```
Comment (dongsug-song) [inline test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/FrameUpdateCallbackTest.cs:371]: could you remove "m" prefix?

File: src/Tizen.NUI/src/internal/Interop/Interop.FrameUpdateCallbackInterface.cs (hunk 1 above)
Comment (bshsqa) [reply src/Tizen.NUI/src/internal/Interop/Interop.FrameUpdateCallbackInterface.cs]: Thank you I fixed it.

File: src/Tizen.NUI/src/internal/Interop/Interop.FrameUpdateCallbackInterface.cs (hunk 2 above)
Comment (bshsqa) [reply src/Tizen.NUI/src/internal/Interop/Interop.FrameUpdateCallbackInterface.cs]: Thank you I fixed it.

File: src/Tizen.NUI/src/public/FrameUpdateCallbackInterface.cs (hunk 3 above)
Comment (bshsqa) [reply src/Tizen.NUI/src/public/FrameUpdateCallbackInterface.cs]: Thank you I fixed it.

File: test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/FrameUpdateCallbackTest.cs (hunk 4 above)
Comment (bshsqa) [reply test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/FrameUpdateCallbackTest.cs]: Thank you I fixed it.

**Review by dongsug-song — APPROVED** [review]
looks good to me

**Pull request merged**